cattle/plugins/docker/compute.py

```diff
@@ -192,13 +192,54 @@ def _is_instance_active(self, instance, host):
 
     @staticmethod
     def _get_docker_client(host):
-        cluster_connection = ""
+        cluster_connection = None
+        tls_config = None
         try:
             cluster_connection = host['clusterConnection']
+            account_id = host['accountId']
+            ca_crt = host['caCrt']
+            client_crt = host['clientCrt']
+            client_key = host['clientKey']
+
```

Comment:
I'll add this check to make sure either all 3 are populated or none is populated.